tests/system.rs

```diff
@@ -75,7 +75,7 @@ fn checkstyle_test() {
 // to a known output file generated by one of the write modes.
 fn assert_output(source: &Path, expected_filename: &Path) {
     let config = read_config(source);
-    let (file_map, _report) = format_file(source, &config);
+    let (_error_summary, file_map, _report) = format_file(source, &config);
```

Comment:
I'm guessing we'll want to adapt a similar "fail on parse error" approach here, but the code path was overall pretty different from the other tests I looked at. Let me know.

Reply:
I don't think we need to check the output here, since we are checking the output of the idempotence tests, which should be the same as the output here.